### PR TITLE
UIDEXP-211: Display title only for instances on error logs page

### PR DESCRIPTION
**Pull request by ViktorSoroka07**
## Purpose

Display title only for instances on error logs page in the scope of the [UIDEXP-211](https://issues.folio.org/browse/UIDEXP-211).

## Jest coverage
Screen Shot 2020-12-15 at 3 14 41 PM

### CODE REVIEW

File: src/components/ErrorLogsView/utils/generateAffectedRecordInfo.js [hunk 1]
```diff
@@ -1,30 +1,42 @@
 import { capitalize } from 'lodash';
 
-function populateAffectedRecords(records, result) {
+import { FOLIO_RECORD_TYPES } from '@folio/stripes-data-transfer-components';
+
+const populateAffectedRecords = (records, result) => {
```
Comment (yelyzavetakhokhlova) [inline src/components/ErrorLogsView/utils/generateAffectedRecordInfo.js:5]: I love arrow functions.

File: src/settings/MappingProfiles/DuplicateMappingProfileRoute/DuplicateMappingProfileRoute.test.js [hunk 2]
```diff
@@ -81,7 +81,6 @@ describe('DuplicateMappingProfileRoute', () => {
     });
 
     it('should have enabled save button if there are no changes', () => {
-      screen.debug(screen.getByRole('button', { name: 'Save & close' }));
```
Comment (yelyzavetakhokhlova) [inline src/settings/MappingProfiles/DuplicateMappingProfileRoute/DuplicateMappingProfileRoute.test.js:84]: whoa, I wonder who left this here...

**Review by yelyzavetakhokhlova — APPROVED** [review]

File: src/settings/MappingProfiles/DuplicateMappingProfileRoute/DuplicateMappingProfileRoute.test.js (hunk 2 above)
Comment (ViktorSoroka07) [reply src/settings/MappingProfiles/DuplicateMappingProfileRoute/DuplicateMappingProfileRoute.test.js:84]: But I eventually removed it. I tracked it down. By the way who removed the PR and allowed it to slip in?🙂

File: src/components/ErrorLogsView/utils/generateAffectedRecordInfo.js (hunk 1 above)
Comment (ViktorSoroka07) [reply src/components/ErrorLogsView/utils/generateAffectedRecordInfo.js:5]: Special for you really😀

File: src/settings/MappingProfiles/DuplicateMappingProfileRoute/DuplicateMappingProfileRoute.test.js (hunk 2 above)
Comment (yelyzavetakhokhlova) [reply src/settings/MappingProfiles/DuplicateMappingProfileRoute/DuplicateMappingProfileRoute.test.js:84]: Haha! I thought it was me, to be honest 😂

Comment (ViktorSoroka07) [reply src/settings/MappingProfiles/DuplicateMappingProfileRoute/DuplicateMappingProfileRoute.test.js:84]: 😬

**Pull request merged**